.github/workflows/ci_check_pr_comment.yml

```diff
@@ -0,0 +1,40 @@
+name: CI PR Comment
+
+# Controls when the action will run.
+on:
+  issue_comment:
+    types: [ created ]
+
+jobs:
+  # reactivecircus/android-emulator-runner@v2 requires MacOS to run on
+  # https://github.com/ReactiveCircus/android-emulator-runner
+  qa_android_tests:
+    # This job only runs for pull request comments and when the comment starts with `/instrumentedTests`
+    if: contains(github.event.comment.html_url, '/pull/') && startsWith(github.event.comment.body, '/instrumentedTests')
```

Comment:
So this will run when our url contains `/pull/` as well?

Reply:
We are basically checking if the comment is for a pull request or not.